runtime/j9vm/j7vmi.c

```diff
@@ -2559,23 +2560,13 @@ jvmDefineClassHelper(JNIEnv *env, jobject classLoaderObject,
 	vmFuncs->internalEnterVMFromJNI(currentThread);
 
 	/* Allocate and initialize a UTF8 copy of the Unicode class-name */
-	if (className == NULL) {
-		utf8Length = 0;
-		utf8Name = NULL;
-	} else {
-		utf8Length = vmFuncs->getStringUTF8Length(currentThread, J9_JNI_UNWRAP_REFERENCE(className));
+	if (className != NULL) {
```

Comment:
Pls put rvalue on the left for inequality comparison.

Reply:
Fixed in 1baa27d